CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.9.2](https://github.com/haskell-nix/hnix/compare/0.9.1...0.9.2) (yyyy-mm-dd)
+
+* Dependencies:
+  * [Update `data-fix` dependency to `>= 0.3.0 && < 0.4`](https://github.com/haskell-nix/hnix/pull/686)
+  * [Update `prettyprinter` dependency to `>= 1.7.0 && < 1.8`](https://github.com/haskell-nix/hnix/pull/679)
+
```

Comment:
Please follow the established structure.

To be understandable and explicit - state that this is a non-breaking change. It may be reasoned from the minor version number, but this style breaks the consistency of the changelog structure. During scrolling/reading through the person observes structure inconsistency.